src/components/Log_Menu.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
+import { log_levels } from './Log_Levels';
+
+interface Props {
+  logFilterValue: number; 
+  onLogFilterChange: (newLogFilterValue: number) => void; 
+}
+
+const BasicSelect: React.FC<Props> = ({ logFilterValue, onLogFilterChange }) => {
+  const handleChange = (event: SelectChangeEvent) => {
+    const newLogFilterValue = event.target.value as unknown as number;
+    onLogFilterChange(newLogFilterValue); 
+  };
+
+  return (
+    <Box sx={{ minWidth: 120 }}>
+      <FormControl fullWidth>
+        <InputLabel id="log-filter-id">Minimum Log Filter</InputLabel>
+        <Select
+          labelId="log-filter-id"
+          id="log-filter-label"
+          value={logFilterValue.toString()}
+          label="Minimum Log Filter"
+          onChange={handleChange}
+        >
+          {Object.entries(log_levels).map(([level_value, level_name]) => (
+            <MenuItem key={level_value} value={parseInt(level_value)}>
+              {level_name}
+            </MenuItem>
+          ))}
```

Comment:
@TBThomas56 I am happy to merge this but in general I would prefer to keep as much logic on the graylog server as possible. I think it should be possible to filter by asking graylog to only give us messages that contain the text, rather than downloading all the messages and then filtering here. I don't mind if you do that in this PR or raise another issue for it.

Reply:
Oh I see what you mean! I think that this is better done in this PR. I will have a go with that and see what I can find.